File: Javascript/uri1013_OMaior.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split(" ");

var a = parseFloat(lines.shift());
var b = parseFloat(lines.shift());
var c = parseFloat(lines.shift());

var maiorAB = (a + b + Math.abs(a - b)) / 2;

maiorAB = (c + maiorAB + Math.abs(maiorAB - c)) / 2;

console.log(maiorAB + " eh o maior");

File: Python/Uri1013_Omaior.py
linha = input().split()

a = int(linha[0])
b = int(linha[1])
c = int(linha[2])

maior = (a+b+abs(a-b))/2
maior = (c+maior+abs(maior-c))/2

print('{} eh o maior'.format(int(maior)))